wolfcrypt/src/asn.c

```diff
@@ -16675,8 +16675,7 @@ static int DecodeGeneralName(const byte* input, word32* inOutIdx, byte tag,
 
             /* test if no ':' char was found and test that the next two
              * chars are "//" to match the pattern "://" */
-            if (i >= len - 2 || (input[idx + i + 1] != '/' ||
-                                 input[idx + i + 2] != '/')) {
+            if (input[idx + i] != ':' || i == 0 || i == len) {
```

Comment:
Comments need updated to reflect no "//"  with the pattern match, an RFC should be referenced for the change too if possible.

Reply:
Comments updated.